descwl_shear_sims/tests/test_gen_star_masks.py

```diff
@@ -0,0 +1,82 @@
+"""
+copy-paste from my (beckermr) personal code here
+https://github.com/beckermr/metadetect-coadding-sims
+"""
+import numpy as np
+
+from ..gen_star_masks import StarMasks
+from ..lsst_bits import STAR, BLEED
+from ..simple_sim import Sim
+
+
```

Comment:
We should add a test here that the code is repeatable if we call it twice with the same rng seed.

Reply:
added